pyHojaDeCalculo

celiabecerril opened the pull request.

Commit:
Creación hoja de calculo(matriz)

celiabecerril committed:
Añade menú interacción con usuario

Con el método interactuarConUsuario() podrá moverse a otras celdas, editar su contenido o salir del programa, a través del teclado. 
w: arriba, a: derecha, s: abajo, d: izquierda, e: editar, q: salir

celiabecerril committed:
Añade lógica de entrada por teclado 

(procesarEntrada(String input, Scanner scanner)) Es el método que se encarga de realizar todas las operaciones de movimiento o edición en la hoja de cálculo

celiabecerril committed:
Estética Hoja de calculo

Añadimos límites entre las celdas, (Esto ha supuesto cambios en métodos ya implementados previamente, (mostrarMatriz(String limite)). Damos formato al programa eliminando espacios y asegurando tabulaciones.

mmasias commented on src-PRG1/HojaCalculo.java:
¿Podemos hacer los identificadores en inglés? Lo digo por ir dejando ya definida esa costumbre

mmasias commented on src-PRG1/HojaCalculo.java:
```diff
@@ -0,0 +1,98 @@
+    private static final int NUM_COLUMNAS = 11;
+    private static String[][] matriz = new String[NUM_FILAS][NUM_COLUMNAS];
+    private static int filaAct = 1;
+    private static int colAct = 1;
```
Pienso en voz alta: ¿y si simplemente le llamamos `FILAS` (o `ROWS`), no quedaría ya claro que es el número de filas? Y fila actual sería `row`, no?

celiabecerril committed:
Creacion matriz

celiabecerril committed:
Añade menú interacción con usuario

Con el método interactWithUser() podrá moverse a otras celdas, editar su contenido o salir del programa, a través del teclado. 
w: arriba, a: derecha, s: abajo, d: izquierda, e: editar, q: salir

celiabecerril committed:
Añade lógica de entrada por teclado

celiabecerril committed:
Estética hoja de cálculo 

Añadimos límites entre las celdas, (Esto ha supuesto cambios en métodos ya implementados previamente, (showMatrix(String limite)). Damos formato al programa eliminando espacios y asegurando tabulaciones.

celiabecerril committed:
Mejoras

Bucle While gobernado por una variable de estado (isRunning)
Creación de método editCell 
Opción "q" (salir) es una constante 
Main en las primeras líneas del programa
